_guide/onboarding-staff.md

```diff
@@ -117,14 +117,14 @@ Onboarding Quiz (Internal) - [Edit view](https://docs.google.com/forms/d/1O7VV0Y
   * All of the above
 * Attach a screenshot of a WAVE or Accessibility Insights report of a government website.
 * What is a best practice to engage with clients and team members that have disabilities?
-  * Always speak loudly and make sure to over emphasize with your lips
+  * Always speak loudly and make sure to overemphasize with your lips
   * With deaf people, look at the interpreter, not the person who you are talking with
   * Ask the person if there is anything you can do to communicate with them more effectively
   * All of the above
 
-### Trello card - Join the #accessibility slack channel & report your score
+### Trello card - Join the #accessibility Slack channel & report your score
 
-We want everyone at CivicActions to get an 80% or higher in this quiz. If you don't score high enough, feel free to engage with the Accessibility Team or retake the exam.
+We want everyone at CivicActions to get an 80% or higher in this quiz. If you don't score high enough, feel free to engage with the Free Culture Team.
```

Comment:
@mgifford can you revert this change as well?

Reply:
You can't get in the way of free culture Daniel!